### DISCUSSION TITLE
Why ColumnConfiguration doest'n have attributesCallback?

### DISCUSSION THREAD

**Discussion by lucassmacedo** [rappasoft/laravel-livewire-tables#1068] [Q&A] [answered]
i'm trying to set the width of a column and looking insde the code i realize that the ColumnConfiguration doest'n have the attributesCallback to set the Width. 
There's some explanation?

**Comment by lrljoe** [comment]
It's in the configure area. 

You can use the setTdAtrributes to do this.

For example:
```php
   $this->setTdAttributes(function (Column $column, $row, $columnIndex, $rowIndex) {
            if ($column->getTitle() == 'Actions') {
                return [
                    'default' => false,
                    'class' => 'w-12',
                ];
            }
        });
```

**Reply by lrljoe** [reply]
Does that make sense to you?

**Comment by lucassmacedo** [comment, marked as answer]
Thanks.

**Reply by lrljoe** [reply]
If this is resolved, then please mark this as answered so that I can help others more easily, and others can find solutions to their issues through searching.

**Reply by lucassmacedo** [reply]
Of course,  where i can mark as answered?
